web/src/features/fba/components/viz/CriticalHours.tsx

```diff
@@ -0,0 +1,18 @@
+import { Typography } from '@mui/material'
```

Comment:
Looks like this file is formatted with 4 spaces for indents isntead of 2.

Reply:
Reran prettier on this file in: https://github.com/bcgov/wps/pull/3909/commits/6a2c057097a689204cd684a411e253f8226f6e05 

We should get it configured to run on save